### PR TITLE
feat: linear.app auth, standard object syncs

### PR DESCRIPTION
**Pull request by tomkit**
## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: docs/docs/providers/linear.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+| ----------------------- | --------- |
+| Authentication (Oauth2) | Yes       |
+| Managed syncs           | Yes       |
+| Actions API             | No        |
+| Real-time events        | No        |
+| Passthrough API         | Yes       |
+
+Supported Standard Objects: `issues`, `comments`, `teams`, `projects`, `users`.
+
+Scopes: `read`, `write`, `issues:create`, `comments:create`. `admin` for full privileges.
```
Comment (lucasmarshall) [inline docs/docs/providers/linear.mdx:22]: This is a little confusing. Some people may read that `admin` is required, but it shouldn't be, right?

File: packages/core/remotes/impl/linear/index.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  clientId: string;
+  clientSecret: string;
+};
+
+class LinearClient extends AbstractNoCategoryRemoteClient {
+  readonly #client: NativeLinearClient;
+  readonly #config: LinearClientConfig;
+  public constructor(config: LinearClientConfig) {
+    super('https://api.linear.app/graphql');
+    const { accessToken } = config;
```
Comment (lucasmarshall) [inline packages/core/remotes/impl/linear/index.ts:29]: Does this expire and need regular renewal? If so, can we implement that like we do in the other remotes?

**Review by lucasmarshall — CHANGES_REQUESTED** [review]

File: packages/core/remotes/impl/linear/index.ts (hunk 2 above)
Comment (lucasmarshall) [reply packages/core/remotes/impl/linear/index.ts:29]: BTW, my preferred implementation of this is in the dynamics remote

Comment (tomkit) [reply packages/core/remotes/impl/linear/index.ts:29]: I took a look: they don't have token refreshing. They just set a really long expiry (10 yrs)

**Review by lucasmarshall — APPROVED** [review]

**Pull request merged**